content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1052 @@
+* **High level data-driven API**: this made it easy and ergonomic to write custom per-entity render logic
+
+However it also had a number of _significant_ shortcomings:
+
+* **Complex**: The "high level ease of use" came at the cost of significant implementation complexity, performance overhead, and invented jargon. Users were often overwhelmed when trying to operate at any level but "high level". When managing "render resources", it was easy to do something "wrong" and hard to tell "what went wrong".
+* **Often slow**: Features like "sprite rendering" were built on the costly high level abstractions mentioned above. Performance was ... suboptimal when compared to other options in the ecosystem.
+* **User-facing internals**: It stored a lot of internal render state directly on each entity. This took up space, computing the state was expensive, and it gunked up user-facing APIs with a bunch of "do not touch" render Components. This state (or at least, the component metadata) needed to be written to / read from Scenes, which was also suboptimal and error prone.
+* **Repeating render logic was troublesome**: Viewports, rendering to multiple textures / windows, and shadow maps were possible, but they required hard-coding, special casing, and boilerplate. This wasn't aligned with our goals for modularity and clarity.
+
+### Why now?
+
+The shortcomings above were acceptable in Bevy's early days, but were clearly holding us back as Bevy grew from a [one person side project](bevyengine.org/news/introducing-bevy) to the most popular Rust game engine on Github (and one of the most [popular open source game engine ... period](https://github.com/topics/game-engine)). A "passable" renderer no longer cuts it when we have hundreds of contributors, a paid full time developer, thousands of individual users, and a growing number of companies paying people to work on Bevy apps and features. It was time for a change.
+
+For a deeper view into our decision making and development process (including the alternatives we considered) check out the [New Renderer Tracking Issue](https://github.com/bevyengine/bevy/issues/2535). 
+
+### Pipelined Rendering: Extract, Prepare, Queue, Render
+
+<div class="release-feature-authors">authors: @cart</div>
+
+Pipelined Rendering is a cornerstone of the new renderer. It accomplishes a number of goals:
+
+* **Increased Parallelism**: We can now start running the main app logic for the next frame, while rendering the current frame. Given that rendering is often a bottleneck, this can be a huge win when there is also a lot of app work to do.
+* **Clearer Dataflow and Structure**: Pipelining requires drawing hard lines between "app logic" and "render logic", with a fixed synchronization point (which we call the "extract" step). This makes it easier to reason about dataflow and ownership. Code can be organized along these lines, which improves clarity.
+
+From a high level, traditional "non-pipelined rendering" looks like this:
```

Comment:
I added comments hyphenating high-level, but just wanted to note that grammatically, this one should not be hyphenated.